internal/test/files.go

```diff
@@ -37,29 +38,50 @@ func AssertFilesEquals(t *testing.T, gotPath, wantPath string) {
 
 func AssertContentToFile(t *testing.T, gotContent, wantFile string) {
```

Comment:
This should be mostly the same, I just extracted the `diff` part to a separate method to reuse it